data/campaigns/Winds_of_Fate/scenarios/08_Foothills.cfg

```diff
@@ -0,0 +1,871 @@
+            [note]
+                description= _ "Flee Gorlack and Reshan through the passage last.
+All those left behind them will perish fighting to buy the rest of us more time.
+Unrecalled veterans will safely return to the Eyrie as they are not present for this crisis."
+            [/note]
+            {HAS_NO_TURN_LIMIT}
+        [/objectives]
+    [/event]
+
+    # Hack to prevent inanimate gate units from turning to face their attacker.
+    [event]
+        name=attack_end
+        first_time_only=no
+        [filter_second]
+            type=Gate
+        [/filter_second]
+        [modify_unit]
+            [filter]
+                type=Gate
+            [/filter]
+            facing=se
+        [/modify_unit]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Arinexis
+            message= _ "To command ressspect, our alliance with dragonsss mussst have <b>ssome</b> name!"
+        [/message]
+        [message]
+            speaker=Zedrix
+            message= _ "‘Sibblingss of Ssetting Sssun’"
+        [/message]
+        [message]
+            speaker=Arinexis
+            message= _ "‘Union of Saurgrath and Morogor’"
+        [/message]
+        [message]
+            speaker=Zedrix
+            message= _ "‘Alignment of Moon and Sssun’"
+        [/message]
+        [message]
+            speaker=Arinexis
+            message= _ "‘Sandwalkersss’"
+        [/message]
+        [message]
+            speaker=Zedrix
+            message= _ "‘Alliance of Sseven Ssilver Starsss and Five Fish of Firmament’"
+        [/message]
+        [message]
+            speaker=Arinexis
+            message= _ "‘Sandfolksss’!"
+        [/message]
+        [message]
+            speaker=Zedrix
+            message= _ "Ah, that is mosst favored name - ‘Sandfolkss’ - bland yet no negative connotationss which sssummon..."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "You <b>will</b> be silent."
+        [/message]
+        {MODIFY_UNIT id=Reshan facing ne}
+        [unhide_unit]
+            id=Reshan
+        [/unhide_unit]
+        [scroll_to_unit]
+            id=Reshan
+        [/scroll_to_unit]
+        [delay]
+            time=1000
+        [/delay]
+        [move_unit]
+            id=Reshan
+            to_x,to_y=17,28
+        [/move_unit]
+        [message]
+            speaker=Gorlack
+            message= _ "Report, Reshan.
+There are still three wings yet to regather."
+        [/message]
+        [message]
+            speaker=Reshan
+            message= _ "Gorlack, the wings are routed.
+
+The jungle withheld the elves’ true number.
+Just as our pride withheld their true potency.
+
+A full throng of them now pursues us here.
+We must retreat to the eyrie with all haste."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "Our haul is too weighty to be flown with a dozen trips.
+
+We can not forsake it.
```

Comment:
'can not' should be cannot

Reply:
This is actually meant as a drakism, that they never use the contraction cannot. Though perhaps that is odd given they use some other formal contractions like none, never and another.

Reply:
I'm not sure I understand you. Cannot isn't a contraction of can not, the two have different meanings, at least in modern contexts. Can not would normally only be used when the 'not' is part of a separate phase after the 'can'. I'm not so sure it's an archaism but if it is, it's potentially very confusing to a modern audience, even one that's expecting an older style of writing.

Reply:
As I understand it, "cannot" is only a _rule_ in modern british dialects of english. In other germanic languages, old english and many north american english dialects, it either does not exist or is an optional contraction. A bit of an inconsistency too, since there is no shallnot, willnot, maynot, etc.

Reply:
The problem is English is full of inconsistencies like that. ;)

While I understand we generally go for older style of writing in Wesnoth - eg Sceptre of Fire retains the older English/French spelling rather than its more modern American variation even in the default en_US locale, and I've followed existing guidelines to retain -ize suffixes rather than follow more modern -ise suffixes as influenced from the French (my personal preference) for en_GB locale - I'm wondering if this is one of those cases where we should forgo that for modern readers. Also those examples I give already have established precedent albeit with regional variations - I'm pretty sure 'can not' in place of 'cannot' is universally considered incorrect.

As a native Anglophone I can deduce things that normally wouldn't be acceptable in English - like the odd capitalisation - may be characteristic of how you want drakes to sound/write. But in this case I was clearly confused by the oddity (I had no idea it was historically acceptable) and I suspect many other modern readers will be too - after all, we can't expect all that many of players would be literary historians.

**tl;dr** If the overall goal is to give drakes an archaic characterisation in their speech/text, in this particular case I think the goal is missed because of confusion/misunderstanding will take precedent over knowledge of historical literary conventions.

Reply:
How would you feel about the 'not' being moved to end of these sentences? Using the above example:

_"We can forsake it not."_

Reply:
That works for me in this case. If you think this syntax can work for the other 'can not' phrases too, I'd be happy with that. Sorry to put more work on you.

Reply:
My above suggestion was _"We can forsake it not."_ (I deleted a duplicate of my comment and github also deleted the original.)

Perhaps _"We must not forsake it."_ might be better anyway.

Reply:
Changed to:
"We must not forsake it.
Too much depends on it."